packages/manager/modules/cda/src/translations/Messages_es_US.json

```diff
@@ -1,12 +1 @@
-{
-  "ceph_common_error": "Lo sentimos, se ha producido un error:",
-  "cda_tab_home": "Resumen",
-  "cda_tab_user": "Utilisateurs",
-  "cda_tab_ip": "Control de acceso IP",
-  "cda_tab_pool": "Pools",
-  "cda_guide_title": "Guías",
-  "cda_guide_name": "Cloud Disk Array (CDA)",
-  "cda_guide_footer": "Todas las guías de OVH",
-  "cda_new_window": "Nueva ventana",
-  "cda_guide_retrieval_error": "Se ha producido un error al cargar la guía sobre Cloud Disk Array (CDA)."
-}
+{}
```

Comment:
why we keep a trad file with no key ? :/

Reply:
This PR has been auto generated, maybe we should improve and delete file if empty